pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -175,6 +175,13 @@ def test_corr_int(self):
         df3.cov()
         df3.corr()
 
+    @td.skip_if_no_scipy
+    def test_corr_axes(self):
+        # https://github.com/pandas-dev/pandas/issues/35002
+        df = pd.DataFrame(np.random.normal(size=(10, 2)))
+        for meth in ["pearson", "kendall", "spearman"]:
```

Comment:
Can you parametrize this instead of using a for loop? May also be worth including a callable method.